svc-aggregation/system/addplugin_test.go

```diff
@@ -250,6 +260,18 @@ func TestExternalInterface_Plugin(t *testing.T) {
 				StatusCode: http.StatusConflict,
 			},
 		},
+		{
+			name: "Adding plugin with duplicate managre uuid",
+			p:    p,
+			args: args{
+				taskID:     "123",
+				req:        reqPluginWithDuplciateUUID,
+				cmVariants: getConnectionMethodVariants("Compute:BasicAuth:STGtest_v1.0.0"),
+			},
+			want: response.RPC{
+				StatusCode: http.StatusCreated,
```

Comment:
shouldn't it be a conflict?

Reply:
Yes, but I have not yet done code changes. Once I do the code changes I will change it to  http.StatusConflict